sql/item_mods.sql

```diff
@@ -20282,13 +20362,17 @@ INSERT INTO `item_mods` VALUES (16808,68,3);
 INSERT INTO `item_mods` VALUES (16809,23,14);
 INSERT INTO `item_mods` VALUES (16810,23,12);
 INSERT INTO `item_mods` VALUES (16816,431,1);
+INSERT INTO `item_mods` VALUES (16817,431,1);   -- Holy Degen +1 - Additional effect: Light damage
 INSERT INTO `item_mods` VALUES (16818,23,5);
 INSERT INTO `item_mods` VALUES (16819,9,3);
 INSERT INTO `item_mods` VALUES (16820,11,3);
 INSERT INTO `item_mods` VALUES (16821,110,10);
 INSERT INTO `item_mods` VALUES (16822,5,10);
 INSERT INTO `item_mods` VALUES (16822,12,5);
+INSERT INTO `item_mods` VALUES (16823,431,1);   -- Flame Degen +1 - Additional effect: Fire damage
+INSERT INTO `item_mods` VALUES (16824,431,1);   -- Carnage Rapier - Additional effect: HP Drain
```

Comment:
HP drain is type 4 instead of type 1, and otherwise needs all the same mods an elemental damage effect would get. Element is darkness.